docs/getting-started/install/kubernetes.rst

```diff
@@ -33,28 +33,6 @@ although you will need to make modifications to the deployment manifests.
    # Create a namespace to run cert-manager in
    kubectl create namespace cert-manager
 
-As part of the installation, cert-manager also deploys a
-`ValidatingWebhookConfiguration`_ resource in order to validate that the
-Issuer, ClusterIssuer and Certificate resources we will create after
-installation are valid.
```

Comment:
It may be worth keeping this paragraph just so users know? & maybe extending it to mention the `APIService` resource too, and mention mutation?

We should also call out conversion too when we have it, and mention it requires 1.15+ (but that can come later)